### ISSUE TITLE
Always run difftool within git dir?

### ISSUE THREAD

**Issue by AeliusSaionji** [Canop/broot#698]
broot runs the following command for `gd`

```git
git difftool -y /home/aelius/qmk_firmware/keyboards/ergodox_ez_rhalf/keymaps/ergopad/
```

But unless I started broot from within qmk_firmare, this will fail with
```
fatal: difftool requires worktree or --no-index
```

Perhaps this is a knowledge deficit on my end, but I did not know that this was a requirement and did not immediately understand what the error was telling me.

If you think it appropriate, perhaps broot could intelligently execute gd and other relevant commands within the git worktree, regardless of where broot was started.

Thanks!

**Comment by Canop** [comment]
You're right. Never noticed that as I always launch this command in a broot launched in a repository.
